### PR TITLE
only prompt if expected copy value is more than the cost of a pocket wish

### PR DESCRIPTION
**Pull request by lewismd13**

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -261,14 +261,15 @@ function getEmbezzlerFight(): EmbezzlerFight | null {
...
-
+  const averageEmbezzlerNet = ((baseMeat + 750) * meatDropModifier()) / 100;
   if (potential > 0) {
     print(`You have the following embezzler-sources untapped right now:`, "blue");
     embezzlerSources
       .filter((source) => !source.available() && source.potential() > 0)
       .map((source) => `${source.potential()} from ${source.name}`)
       .forEach((text) => print(text, "blue"));
     if (
+      Math.floor((potential * averageEmbezzlerNet) / 50000) > 0 &&
```
Comment (horrible-little-slime) [inline src/fights.ts:272]: This feels like a profoundly weird way to do the math on this--is there a reason you've chosen to execute it this way?

File: src/fights.ts [hunk 2]
```diff
@@ -261,14 +261,15 @@ function getEmbezzlerFight(): EmbezzlerFight | null {
     if (fight.available()) return fight;
   }
   const potential = embezzlerCount();
-
+  const averageEmbezzlerNet = ((baseMeat + 750) * meatDropModifier()) / 100;
```
Comment (horrible-little-slime) [inline src/fights.ts:264]: Per discussion in discord, this is weird but fine for now

File: src/fights.ts (hunk 1 above)
Comment (lewismd13) [reply src/fights.ts:272]: I swear it made sense in my head at that the time. We can just do potential * averageEmbezzlerNet >50000 I suppose

Comment (horrible-little-slime) [reply src/fights.ts:272]: Make it so and I'll push it

File: src/fights.ts [hunk 3]
```diff
@@ -261,14 +261,15 @@ function getEmbezzlerFight(): EmbezzlerFight | null {
...
   if (potential > 0) {
     print(`You have the following embezzler-sources untapped right now:`, "blue");
     embezzlerSources
       .filter((source) => !source.available() && source.potential() > 0)
       .map((source) => `${source.potential()} from ${source.name}`)
       .forEach((text) => print(text, "blue"));
     if (
+      potential * averageEmbezzlerNet > 50000 &&
```
Comment (horrible-little-slime) [inline src/fights.ts:272]: (potential + 1) * (averageEmbezzlerNet - get("valueOfAdventure"))

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]

**Pull request merged**